### PR TITLE
[EAK-299]: Add site analytics, polyfills and extend browser support

### PR DESCRIPTION
**Pull request by ala-n**

### CODE REVIEW

File: docs/website/views/_includes/analytics.njk [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<!-- Global site tag (gtag.js) - Google Analytics -->
+<script async src="https://www.googletagmanager.com/gtag/js?id=G-FTFSXX8LXV"></script>
+<script>
+  window.dataLayer = window.dataLayer || [];
+  function gtag(){dataLayer.push(arguments);}
```
Comment (nattallius) [inline docs/website/views/_includes/analytics.njk:5]: could you add formatting, please?

**Review by nattallius — APPROVED** [review]

File: docs/website/webpack.config.js [hunk 2]
```diff
@@ -4,7 +4,9 @@ module.exports = {
   mode: 'development',
   devtool: 'source-map',
   entry: {
-    'localdev': './src/localdev.ts',
+    'site': './src/site.ts',
+    'polyfill-light': './src/polyfill-light.ts',
+    'polyfill-medium': './src/polyfill-medium.ts'
```
Comment (abarmina) [inline docs/website/webpack.config.js:9]: we don't need `polyfill-full` support?

**Review by abarmina — APPROVED** [review]

File: docs/website/views/_includes/analytics.njk (hunk 1 above)
Comment (ala-n) [reply docs/website/views/_includes/analytics.njk:5]: That's the snippet from Google Tags, but ok

File: docs/website/webpack.config.js (hunk 2 above)
Comment (ala-n) [reply docs/website/webpack.config.js:9]: We do not support IE11 and ES5-time browsers

**Review by NastaLeo — APPROVED** [review]

File: docs/website/webpack.config.js (hunk 2 above)
Comment (abarmina) [reply docs/website/webpack.config.js:9]: this is great news!

**Review by liubou-masiuk — APPROVED** [review]

**Pull request merged**